src/Plugin/Block/ContactsEntity.php

```diff
@@ -178,6 +198,24 @@ public function getEditLink($mode) {
     return $link;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function processManageMode(array &$variables) {
+    $definition = $this->blockManager->getDefinition($variables['id']);
```

Comment:
Can this not just use `$this->definition` and avoid injecting block manager entirely?

Reply:
doh!